@here/harp-mapview/lib/ClipPlanesEvaluator.ts

```diff
@@ -175,6 +179,43 @@ export class AltitudeBasedClipPlanesEvaluator implements ClipPlanesEvaluator {
         return clipPlanes;
     }
 
+    /**
+     * Set maximum elevation to be rendered (intersects frustum) above the sea level.
+     * @param elevation the elevation (altitude) value in world units (meters).
+     * @note Reasonable values are in between (0, MtEverestHeight>, where the second is
+     * defined in [[EarthConstant.MAX_ELEVATION]]
+     */
+    set maxElevation(elevation: number) {
+        assert(elevation >= 0);
+        this.nearMargin = elevation;
+    }
+
+    /**
+     * Get maximum elevation to be covered by camera frustum.
+     */
+    get maxElevation(): number {
+        return this.nearMargin;
+    }
+
+    /**
+     * Set minimum elevation beneth the sea level to be rendered.
+     * @param elevation the minimum elevation in world units (meters), should be greater then 0.
+     * @note If you set this parameter to zero you may not see any features rendered if they are
+     * just at or below the sea level. Reasonable values are between (0, DeadSeaLevel>, where the
+     * second is the lowest depression on the earth defined in [[EarthConstants.MIN_ELEVATION]]
+     */
+    set minElevation(elevation: number) {
```

Comment:
I'm not a fan of having a negative number represented, it is a bit confusing IMO. Is there a strong reason to have it?
Also, there may be cases where the minElevation is < 0, for example in Switzerland, or Chile, there are large areas of the ground where the minElevation would be < 0 (or positive in absolute elevation above sea level)

Reply:
Hmm... good point. The basic idea was to support only depressions with this parameter, so it would be considered as offset from sea level deep to earth center, so we can not have both min/max elevations above the ground. Of course I could revise this to support positive and negative values, but we may also do this in the next patch-set, because it is not the only thing required to fully support elevation feature in the engine.
Give me a note if we want to have it in the same commit.

Reply:
I think the assumption that the sea-level needs always to be rendered should be removed, this is in many cases not the case, and I think it makes sense to do it the right way first and now, rather than having a follow up pull request, however having a separate commit in this pull request is fine.

Reply:
Ok.